app/src/test/java/org/eyeseetea/malariacare/domain/entity/InvalidLoginAttemptsTest.java

```diff
@@ -0,0 +1,97 @@
+package org.eyeseetea.malariacare.domain.entity;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+public class InvalidLoginAttemptsTest {
+
+    @Test
+    public void check_when_creates_login_enable() {
+        InvalidLoginAttempts invalidLoginAttempts = new InvalidLoginAttempts(0, 0);
+        assertThat(invalidLoginAttempts.isLoginEnabled(), is(true));
+    }
+
+    @Test
+    public void check_not_disable_when_add_one_attempt() {
+        InvalidLoginAttempts invalidLoginAttempts = new InvalidLoginAttempts(0, 0);
+        invalidLoginAttempts.addFailedAttempts();
+        assertThat(invalidLoginAttempts.isLoginEnabled(), is(true));
+    }
+
+    @Test
+    public void check_disable_after_add_three_attempts() {
+        InvalidLoginAttempts invalidLoginAttempts = new InvalidLoginAttempts(0, 0);
+        invalidLoginAttempts.addFailedAttempts();
+        invalidLoginAttempts.addFailedAttempts();
+        invalidLoginAttempts.addFailedAttempts();
+        assertThat(invalidLoginAttempts.isLoginEnabled(), is(false));
+    }
+
+    @Test
+    public void check_disable_after_add_three_attempts_and_twenty_seconds()
+            throws InterruptedException {
+        InvalidLoginAttempts invalidLoginAttempts = new InvalidLoginAttempts(0, 0);
+        invalidLoginAttempts.addFailedAttempts();
+        invalidLoginAttempts.addFailedAttempts();
+        invalidLoginAttempts.addFailedAttempts();
+
+
+        Thread.sleep(20000);
```

Comment:
@manuelplazaspalacio unit test should execute in miliseconds, we have find a way for not wait 20 or 30 seconds for test. for example, can we pass interval as parameter?